pages/contributing/metadata_guidelines.md

```diff
@@ -2,3 +2,185 @@
+
+## Metadata attributes of a page
+When creating a new page in RSQKit, the following metadata attributes may be edited:
+
+* `title`: Specify here the title of the page. This wil be the H1 title (replacing the top level title using the # in markdown )
+
+* `summary`: Using this attribute it is possible to specify a summary which will be displayed under the title of the page. This summary will also be used as description of your page when the page is tagged.
+
+* `description`: Short sentence about the page starting with a lowercase. This sentence is visualized when pages are automatically listed using a tag.
+
+* `contributors`: list here all the contributors that helped in establishing the page. This will be the full name of the person. Make sure that the person name that is listed can be found in the CONTRIBUTORS.yaml file in the _data directory if you want to link the github id and other contact information (see below for contributor metadata).
+
+* `search_exclude`: by setting this field true, the page will not end up in the search results of the searchbar. By default this is false.
+
+* `sitemap`: let the page appear in the sitemap.xml. Default: true
+
+* `no_robots`: by setting this field to true, the page will not end up in the search results of google or any other search engine.
+
+* `hide_sidebar`: When true, the sidebar will be hidden. Default: false.
+
+* `custom_editme`: This attribute can be used to specify an alternative file/link when clicked on the edit-me button.
+
+* `keywords`: List here all the keywords that can be used to find the page using the searchbox in the right upper corner of the page, lowercase.
+
+* `sidebar`: Specify here an alternative sidebar. Default: main.
+
+* `toc`: When set to false, the table of contents in the beginning of the page will not be generated.
+
+* `page_id`: Unique identifier of a page. It is usually a shortened version of the page name or title, with small letters and spaces, or an acronym, with capital and small letters. Used to list `Related` pages.
+
+* `datatable`: use this attribute to activate the pagination + sorting + searching in tables
+
+### Related pages
+
+* `related_pages`: List here the `page_id` of {{site.title}} pages that you want to display as Related pages, grouped by section.
+
+  If you want pages from the specific section (Your tasks, Your domain, Tool assembly) to be shown here as Related pages, list their `page_id`. If you want to list multiple related pages, make sure to put them in a list like this: [page_id1, page_id2]. The specific sections allowed in each page are specified in each page template. Please, do not add extra sections in the metadata of the page.
+
+  ```yml
+  related_pages: 
+    - your_tasks: [page_id1, page_id2]
+    - your_domain: [page_id1, page_id2]
+    - tool_assembly: [page_id1, page_id2]
```

Comment:
```suggestion
```

Reply:
There is no such thing as a 'tool_assembly' in RSQKit at the moment

Reply:
I left it because this part I copied from rdmkit. But happy to remove, indeed.